Data.json

```
@@ -169,4 +169,8 @@
     "username": "Heismanish",
     "avatar": "https://github.com/Heismanish.png"
   }
+  {
+    "name": "Likhith K",
+    "username": "likhith4",
+    "avatar": "https://github.com/likhith4.png"
 ]
```

Please add the missing two ending curly brackets.
```suggestion
  }
}
```